from domain.exceptions.domain_exception import DomainException


class CoordinatesException(DomainException):
    pass
